llarp/util/threading.cpp

```diff
@@ -24,6 +30,8 @@ namespace llarp
         LogError("Failed to set thread name to ", name, " errno = ", rc,
                  " errstr = ", strerror(rc));
       }
+
+#endif
```

Comment:
is this a stray endif? why was it aadded?

Reply:
because of the `#else` on line 22.